src/main/webapp/scripts/group-details-script.js

```diff
@@ -142,122 +142,23 @@ function diff(obj1, obj2) {
 /** Function called when user clicks to view access and membership settings */
 function viewSettings() {
     var settingsChart = document.getElementById("group-settings-chart");
-    if (settingsChart.classList.contains("hidden")) {
-        settingsChart.classList.remove("hidden")
-    } else {
-        settingsChart.classList.add("hidden")
-    }
+    var viewSettingsLink = document.getElementById("view-settings-link");
```

Comment:
With change to hide and show Elements, both 145 and 146 can be removed, and you can just have hideElements and showElements take a list of Strings